test_helpers/doer_mock.go

```diff
@@ -0,0 +1,35 @@
+package test_helpers
+
+import (
+	"bytes"
+
+	"github.com/tarantool/go-tarantool/v2"
+)
+
+type DoerResponse struct {
```

Comment:
It looks like an implementation detail.
```suggestion
type doerResponse struct {
```